webapp/src/sagas.js

```diff
@@ -339,4 +341,34 @@ function* handleIntentUnconfirmedBid(action) {
   }
 }
 
+const BID_INCREASE = 1.1
+
+function* handleFastBid(action) {
+  const parcel = action.parcel
+  if (parcel.projectId) {
+    // TODO: Show message, can't bid on project
+    return
+  }
+  const { x, y } = parcel
+  const amount = parcel.amount ? Math.ceil(parcel.amount * BID_INCREASE) : 1000
+  const addressState = yield select(
+    selectors.getAddressState
+  )
+  if (addressState.loading || !addressState.data.balance) {
+    // TODO: Balance not loaded?
+    return
+  }
+  if (amount > addressState.data.balance) {
+    // TODO: Not enough balance
```

Comment:
I think we're missing a return here so this doesn't happen:

![image](https://user-images.githubusercontent.com/692077/33712018-3d73e0c2-db46-11e7-858e-b4a542a940f9.png)

Reply:
Done